config/blazer.yml

```diff
@@ -0,0 +1,73 @@
+# see https://github.com/ankane/blazer for more info
+
+data_sources:
+  main:
+    url: <%= ENV["BLAZER_DATABASE_URL"] %>
```

Comment:
This has been set in production

Reply:
should it be added to the `Envfile` as well? This way people can use it in dev as well.

To test this branch I set it to:

```yml
BLAZER_DATABASE_URL: postgresql://localhost/PracticalDeveloper_development
```

in my local `application.yml` file and it worked

Reply:
I didn't set up anything locally and it worked so I think we can leave this.